restructuredtext_lint/lint.py

```diff
@@ -80,9 +75,24 @@ def error_collector(data):
     return errors
 
 
-def lint_file(filepath, encoding=None):
+def lint(content, filepath=None, ignore_sphinx=True):
```

Comment:
Or something like:

     std = 0
    python = 2
    javascript = 4
    cpp = 8
    c = 16

Then people can xor those together and that will affect what we ignore?